### DISCUSSION TITLE
CTA/Submit alignment

### DISCUSSION THREAD

**Discussion by HartiganHM** [twilio-labs/paste#1323] [Q&A] [answered]
Hello!

We're building out a UI for a "wizard" walkthrough that terminates in a form submission. There will be several "Continue/Next step" buttons throughout the "wizard" as information is gathered and a final "Submit" button when a user fills out a form.

Does Paste have any guidance on how this should be aligned if there is only _one_ primary CTA? I assume left-aligned, but wanted to double check. There are a few screenshots below for reference.

Thanks!
 
 **Wizard**
> Right-alignment
![image](https://user-images.githubusercontent.com/24458700/112687131-a3124e80-8e3c-11eb-85c0-e5bc83225fad.png)
> Left-alignment
![image](https://user-images.githubusercontent.com/24458700/112687157-af96a700-8e3c-11eb-9bfe-c5f8c9bc0822.png)

**Form**
> Right-alignment
![image](https://user-images.githubusercontent.com/24458700/112687252-d81ea100-8e3c-11eb-9200-b47b75bd0c6b.png)
> Left-alignment
![image](https://user-images.githubusercontent.com/24458700/112687282-e2d93600-8e3c-11eb-96f1-ebe65900e6a7.png)

**Comment by corinne-nagel** [comment, marked as answer]
Hi @HartiganHM ! Thanks for sharing this! This pattern is currently being designed for, you can always [check out patterns in progress](Wizard) here.

Would this wizard live in a page layout or a modal?
[Here's an awesome article](https://uxdesign.cc/buttons-placement-and-order-bb1c4abadfcb) that explains button positioning. I think your instincts are correct and the primary action should live bottom left if this wizard is in a page. If it's in a modal, [this page](https://paste.twilio.design/components/modal#footer-actions-alignment) will give you better guidance.

Hope this helps, thanks!

**Reply by HartiganHM** [reply]
It's not in a modal, so it sounds like bottom-left is the winner. Awesome resources too, thanks for the help!
